tests/test_theano.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+import theano
+from theano import tensor
+
+class TestTheano(unittest.TestCase):
+    def test_addition(self):
+        # declare two symbolic floating-point scalars
```

Comment:
Is it typical for github codebases to have sentence comments that aren't capitalized and punctuated?

I'd personally prefer we adopt the internal python style, at least incrementally:
  https://engdoc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head

Reply:
I am fine with that.